newuser/newuser.js

```diff
@@ -0,0 +1,292 @@
+const { Cipher } = require('crypto');
+const fs = require('fs')
+const path = require('path')
+const u = require('@elife/utils')
+const { ipcRenderer } = require('electron');
+const {remote} = require('electron');
+
+const Path = require('path')
+
+var pharseParam ='';
+var pharseParam2=''
+var phrase='';
+var selectrdPhraseArr=[];
+var pubkey;
+var seckey;
+var publickey;
+var secretkey;
+var elifeKeys;
+var secureKeys = '';
+var passMatch= false;
+if (document.URL.split('?')[0].includes('4.html')){
```

Comment:
This should probably be invoked from newuser/4.html instead

i.e. in newsier/4.html script tag add a call to a function `generateKey(document.getElementById('generate'))`  or something similar

Reply:
tried the same but, upon redirection the the previous variable value set is getting cleared on undefined

Reply:
Variables will always get cleared on new pages. That's to be expected. Can you detail the issue a bit more? What variable and how can this solve that problem?

Reply:
Updated as Function